homeassistant/components/tedee/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "tedee",
+  "name": "Tedee",
+  "codeowners": ["@patrickhilker", "@zweckj"],
```

Comment:
@patrickhilker are you fine to be a codeowner if I move this to core, or should I remove you?

Reply:
Awesome - I'm fine beeing a codeowner.